java/src/jmri/jmrix/roco/z21/Z21XNetOpsModeProgrammer.java

```diff
@@ -22,6 +22,8 @@ public class Z21XNetOpsModeProgrammer extends jmri.jmrix.lenz.XNetOpsModeProgram
     private int _cv;
     private LnTrafficController lnTC;
 
+    static public int operationDelay = 50;
```

Comment:
If this is intended to be public to be changeable by scripts, it would be good to have a comment about it. So that no one changes this to private.